Assets/Scripts/Game/MagicAndEffects/Effects/Special/WabbajackEffect.cs

```diff
@@ -64,7 +64,7 @@ public override void MagicRound()
                 DaggerfallEntityBehaviour newEnemyBehaviour = gameObject.GetComponent<DaggerfallEntityBehaviour>();
                 EnemyEntity newEnemy = (EnemyEntity)newEnemyBehaviour.Entity;
                 newEnemy.WabbajackActive = true;
-                newEnemy.CurrentHealth = enemy.MaxHealth - enemy.CurrentHealth; // carry over damage to new monster
+                newEnemy.CurrentHealth -= enemy.MaxHealth - enemy.CurrentHealth; // carry over damage to new monster
```

Comment:
I wonder if this could create a situation where enemy dies instantly because the original enemy has lost more health than the transformed enemy has available? Another possible approach would be to get percent of health lost on original enemy then remove that percent of health from new enemy.

Reply:
That happens pretty often in classic, actually. I was going for behavior that would mimic the original.

Reply:
Sounds good, I honestly wasn't sure what classic's behaviour was here. :)